x-pack/plugins/security_solution/public/timelines/components/flyout/pane/index.tsx

```diff
@@ -46,8 +46,8 @@ const FlyoutPaneComponent: React.FC<FlyoutPaneComponentProps> = ({
           data-test-subj="timeline-flyout"
           css={css`
             min-width: 150px;
-            height: calc(100% - 96px);
-            top: 96px;
+            height: 100%;
+            top: 0;
```

Comment:
```suggestion
            top: 0;
            left: 0;
```

Reply:
Everything else looks great! We just need timeline to extend to the left fully as well 😄

Reply:
Perfect, applied! 👍